Makefile

```diff
@@ -36,7 +45,7 @@ all:  ## Build binaries for all available architectures
 	make crossbuild GOOS=windows GOARCH=386
 
 .PHONY: build_local
-build_local:  ## Build local binaries without providing specific GOOS/GOARCH
+build_local: ## Build local binaries without providing specific GOOS/GOARCH
```

Comment:
We should also add `web` as build_local dependencies because makefile will use `make build_local` by default